prime-dotnet-webapi/Infrastructure/TextSearchQuery.cs

```diff
@@ -34,11 +34,24 @@ public TextSearchQuery<T> SearchCollections(params Expression<Func<T, IEnumerabl
         public IQueryable<T> Containing(string term)
         {
             term = term.ToLower();
+            var searchWords = term.Split(' ');
             var predicate = PredicateBuilder.New<T>();
 
             foreach (var selector in _propertySelectors)
             {
-                predicate.Or(x => selector.Invoke(x).ToLower().Contains(term));
+                // This block allows searching and matching for partial names for example Jo Do when searching for Jone Doe
```

Comment:
So for this user story it is a signing authority phoning in and giving prime support their name to find their admin entry. I think for this case we can assume correct spelling of the name and just use the contains logic that already exists. Since this looks like you would get some weird results. Since you don't differentiate the first and second search term to be first and last name.

So when a admin types "Jo Do" it would return 

John Smith
Don Jeremy
Jone Doe
Nathan Johnson

Reply:
If we use the existing contain, the check won't match anything way more often. Since the two fields are searched independently, writing Jone D won't even find the proper Jone Doe if it already exists

Reply:
Hmm this won't allow for partial searches